aptos-move/aptos-abstract-gas-usage/src/algebra_helpers.rs

```diff
@@ -79,46 +79,48 @@ pub fn expand_terms(expr: DynamicExpression) -> Vec<DynamicExpression> {
 ///
 /// (A + A) * (5 + 5) => 5A + 5A + 5A + 5A => 20A
 pub fn aggregate_terms(terms: Vec<DynamicExpression>) -> Result<BTreeMap<String, u64>> {
+    use DynamicExpression::*;
```

Comment:
Usually it's better to enumerate what you use, so changes don't break your code.

Reply:
agreed, I don't see how this is better.

Reply:
I now manually import all the variants. Thanks for pointing this out!